src/app/story/directives/basic.component.css

```diff
@@ -7,6 +7,20 @@ hr {
   border-bottom: 1px solid #ccc;
   margin: 0 0 30px;
 }
+
+prx-select {
+  position: absolute;
+  top: -4px;
+  right: 0;
+}
+/* TODO: move to styleguide */
```

Comment:
was this moved? (The styleguide has an open issue about making global changed and invalid styles BTW https://github.com/PRX/styleguide.prx.org/issues/30) Bigger problem: I can't seem to get that field to work.